### PR TITLE
Feature/429 membercard component

### PR DESCRIPTION
**Pull request by 0xRowdy**
## GitHub Issue

[GitHub issue](https://github.com/HausDAO/daohaus-monorepo/issues/429)

## Changes

Created MemberCard Dropdown

## Packages Added

NA

## Checks

Before making your PR, please check the following:

- [x] Critical lint errors are resolved
- [x] App runs locally
- [x] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

**Review by skuhlmann — APPROVED** [review]

File: libs/ui/src/components/organisms/MemberCard/MemberCard.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  DropdownLink,
+  DropdownText,
+  DropdownMenuItem,
+  ProfileAvatar,
+} from '../../molecules';
+import { MemberCardTrigger } from './MemberCard.styles';
+import { useToast } from '../../../hooks';
+
+type MemberCardProps = {
+  ceramicProfile: {
```
[the remark this answers is not in the document]
Comment (skuhlmann) [reply libs/ui/src/components/organisms/MemberCard/MemberCard.tsx:24]: Yeah I think this makes sense... or we could just pass the args we need for this. Here is the full AccountProfile type:

```ts
export type AccountProfile = {
  address: string;
  ens?: string;
  image?: string;
  name?: string;
  description?: string;
  emoji?: string;
  lensHandle?: string;
  lensId?: string;
  daos?: ITransformedMembershipsQuery['daos'];
};
```

[the remark this answers is not in the document]
Comment (0xRowdy) [reply libs/ui/src/components/organisms/MemberCard/MemberCard.tsx:24]: Sweet, I think I'll put that into the types folder and reference it there unless you already have it someplace in utill?

[the remark this answers is not in the document]
Comment (skuhlmann) [reply libs/ui/src/components/organisms/MemberCard/MemberCard.tsx:24]: it's in the dao-data lib:

https://github.com/HausDAO/daohaus-monorepo/blob/develop/libs/dao-data/src/types/profile.types.ts

File: libs/ui/src/components/organisms/AppSwitcher/AppSwitcher.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { IconType } from 'react-icons/lib';
+import { RiArrowDropDownLine } from 'react-icons/ri';
+
+import { Dropdown, DropdownMenuItem, DropdownButton } from '../../molecules';
+import { AppSwitcherTrigger } from './AppSwitcher.styles';
+
+type DaoHauseApp = {
```
Comment (skuhlmann) [inline libs/ui/src/components/organisms/AppSwitcher/AppSwitcher.tsx:8]: spelling

File: libs/ui/src/components/organisms/MemberCard/MemberCard.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  DropdownText,
+  DropdownMenuItem,
+  ProfileAvatar,
+} from '../../molecules';
+import { MemberCardTrigger } from './MemberCard.styles';
+import { useToast } from '../../../hooks';
+
+type MemberCardProps = {
+  profile: AccountProfile;
+  explorerNetworkId: keyof Keychain;
```
Comment (skuhlmann) [inline libs/ui/src/components/organisms/MemberCard/MemberCard.tsx:26]: ValidNetworks

File: libs/ui/src/components/organisms/MemberCard/MemberCard.tsx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    if (explorerNetworkId) {
+      return generateExplorerLink({
+        chainId: explorerNetworkId,
+        address: profile.address,
+        type: 'address',
+      });
+    }
+  }, [profile, explorerNetworkId]);
+
+  const handleCopy = () => {
```
Comment (skuhlmann) [inline libs/ui/src/components/organisms/MemberCard/MemberCard.tsx:53]: make a util

File: libs/ui/src/components/organisms/MemberCard/MemberCard.tsx [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+      }
+    >
+      <DropdownMenuItem>
+        <DropdownLink href={`/profile/${profile.address}`}>
+          <ParMd>View Profile</ParMd>
+        </DropdownLink>
+      </DropdownMenuItem>
+      <DropdownMenuItem>
+        <DropdownLink href={explorerLink} linkType="external">
+          <ParMd>{networkName}</ParMd>
```
Comment (skuhlmann) [inline libs/ui/src/components/organisms/MemberCard/MemberCard.tsx:87]: Hard code to:

Block Explorer

**Review by skuhlmann — COMMENTED** [review]
see notes

**Pull request merged**